lib/kender/commands/cypress_rails.rb

```diff
@@ -0,0 +1,16 @@
+module Kender
+  class CypressRails < Command
+
+    def available?
+      in_gemfile?('cypress-rails')
```

Comment:
Instead of making your `command` complex, it seems you should do this:
```suggestion
      in_gemfile?('cypress-rails') && Rake::Task.task_defined?('cypress_extended:enhanced_run')
```